spec/memery_spec.rb

```diff
@@ -134,12 +134,26 @@ def self.macro(name)
   macro memoize def g; end
 end
 
+
 RSpec.describe Memery do
   subject(:a) { A.new }
 
   before { CALLS.clear }
   before { B_CALLS.clear }
 
+  let(:h_nomemo_class) do
```

Comment:
I generally prefer anonymous classes for this sort of testing, because we can control them inside spec contexts. But I see you have a consistent style based around normal constant classes at the top, which I'm happy to adopt if you prefer it.

Reply:
I think we better stick to one style everywhere :)

Reply:
Check, I'll get a revision to your norm up shortly :-)

Reply:
Ooh, there's actually a problem with that here, because I'm writing specs that check the return values of calling memoize _on the class_. Those specs definitely should be operating on an anonymous class, because the spec itself is modifying the class (by applying the decorator).

I think the `h_class` value here can be a constant like you're using above, but not `h_nomemo_class` - would that be alright?

Reply:
(went ahead and made `h_class` into a constant `H` like the others, but renamed `h_nomemo_class` as `unmemoized_class`, since that's a clearer name that doesn't entangle with the letter-based naming of the constants)